cmd/wallet-web/src/layouts/WACI.vue

```diff
@@ -12,14 +12,37 @@
 
 <script>
 import WACIShareForm from '@/pages/WACIShare.vue';
+import WACIIssuanceForm from '@/pages/WACIIssue.vue';
```

Comment:
Can we keep it consistent with the filename `WACIIssueForm` maybe?

Reply:
Issuance & Share are WACI flow names in the WACI specifications.